src/backtrace.h

```diff
@@ -150,7 +150,13 @@ void backtrace_gdb(void) {
     const char fmt[] = "%s -nx -batch -x %s '%s' %d";
     static char cmd[sizeof(fmt) + 3 * MAX_BT_PATHSIZE];
     char filename[MAX_BT_PATHSIZE];
-    const char *gdb = (access(GDB_PATH, X_OK) ? "gdb" : GDB_PATH);
+    char *gdb = NULL;
+    char *backtrace_path = shmem_internal_params.BACKTRACE_PATH;
+    if (0 == strcmp(backtrace_path, "")) {
+        gdb = (access(GDB_PATH, X_OK) ? "gdb" : GDB_PATH);
+    } else {
+        gdb = (access(backtrace_path, X_OK) ? "gdb" : backtrace_path);
+    }
```

Comment:
No, these just get the URL based on the paths provided. If the permission does not allow from `access` then just `gdb` is added as the URL since we already defined `USE_BT_GDB`.